libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -675,6 +675,18 @@ namespace eosio { namespace testing {
      string expected;
   };
 
+  /**
+   * Utility predicate to check whether an fc::exception message contains a given string
+   */
+  struct fc_exception_message_contains {
+     explicit fc_exception_message_contains( const string& msg )
+           : expected( msg ) {}
```

Comment:
Better to do:

```
explicit fc_exception_message_contains( string msg )
           : expected( std::move(msg) ) {}
```

so that if a rvalue is passed it is moved.

Reply:
Done.